tests/tests/test_fault_tolerance.py

```diff
@@ -257,10 +248,6 @@ def test_rootfs_conf_missing_from_new_update(
 
         mender_device = standard_setup_one_client_bootstrapped.device
 
-        output = mender_device.run("test -e /data/mender/mender.conf", hide=True)
-        if output.return_code != 0:
-            pytest.skip("Needs split mender.conf configuration to run this test")
```

Comment:
This should probably be retained with an `&& echo true` so you can check the output. Otherwise we can't run this with thud and older.